Sources/MeiliSearch/Config.swift

```diff
@@ -11,8 +11,8 @@ public class Config {
 
     // MARK: Static
 
-    ///Deafault configuration for the default MeiliSearch host, do not use this in
-    ///production since it does not contains the apiKey.
+    /// Deafault configuration for the default MeiliSearch host, do not use this in
```

Comment:
```suggestion
    /// Default configuration for the default MeiliSearch host, do not use this in
```
( nothing to do with your PR but why not hihi)